src/core.mk

```diff
@@ -61,7 +61,8 @@ INSTALL_HEADERS = legate.h                        \
 									legate_preamble.h               \
 									core/legate_c.h                 \
 									core/comm/communicator.h        \
-									core/comm/coll.h		\
+									core/comm/coll.h		        \
```

Comment:
can you align this backslash with the others?

Reply:
Hmm, it looked aligned on my editor. Let me take a look again